app/controllers/api/v1/downloads_controller.rb

```diff
@@ -44,6 +44,26 @@ def batch_export
         download(storage, file_location)
       end
 
+      # GET /downloads/batch_import/:id(/:option)
+      def batch_import
```

Comment:
@cgalarza Could you please add a test for this in downloads_spec.rb?

Reply:
Sure!

Reply:
Testing this definitely caught at least one problem. Thanks for the reminder.

Reply:
👍